scripts/west_commands/sign.py

```diff
@@ -350,12 +359,17 @@ def edt_flash_params(flash):
         if align == 0:
             log.die('expected nonzero flash alignment, but got '
                     'DT flash device write-block-size {}'.format(align))
-        reg = image_0.regs[0]
+        reg = images['image-0'].regs[0]
+        if reg.addr == 0:
+            log.die('expected nonzero slot address, but got '
+                    'DT image-0 partition address {}'.format(reg.addr))
+        addr = reg.addr
```

Comment:
This change seems unrelated to using image-1's size. Why is it being added?

Reply:
This was just added because `addr` and `size` in that commit changed to arrive from different places, and it seemed like a good idea to test it then, but in retrospect since `addr` is not used by imgtool, only by printing routines, there is not point in testing it, so I removed this in the fixed commit.